### ISSUE TITLE
From twitter "I like software but I love systems."  What kind of systems?

### ISSUE THREAD

**Issue by hankroark** [jasoncwarner/ama#32]
Techincal, social, socio-technical?  All of the about?  How do you go about building "systems"?

**Comment by jasoncwarner** [comment]
The types I have found I really like in my career, probably in order of either preference or skill I seem to have in the system

- industry/sector evolution 
- company mission/vision/strategy 
- company building
- distributed systems architectures

As you might see from my list, I tend to like or have more aptitude for larger, more complex, longer time horizons, and likely less specific and detailed systems. I think this goes to the way I think and engage in topics. I am a classic T shape generalist person when it comes to subjects. 

I would say I'm a generalist in:

- all things software development, software engineers, 
- technical industry and trends
- software trends themselves


I would say I'm a deep expert in:

- leadership
- strategy/vision/company building in products related to dev tools, engineers
- platforms/cloud
- bottoms up product led growth



There are many other types of systems that can be considered and should. There are closed systems, open systems, sub-systems etc etc etc. Here are some examples:

- database schema
- application architectures
- team or teams or division
- company department
- open source community
- political campaign
- a school etc

Basically, whatever the *largest* grouping of "things" that I had purview into at the time was how I gravitated. I have come to really like when there are multiple interconnected systems at play and working through them.
